test/google_search_data_viewer/keyword/search_results_test.exs

```diff
@@ -54,4 +54,108 @@ defmodule GoogleSearchDataViewer.Keyword.SearchResultsTest do
              }
     end
   end
+
+  describe "group_by_adword_types/1" do
+    test "given SearchResultUrls with two URLs that are not adwords, returns a Map with two non_adwords with the key non_adwords" do
+      keyword_upload = insert(:keyword_upload)
+
+      search_results =
+        insert_list(2, :search_result_url,
+          is_adword: false,
+          is_top_adword: false,
+          keyword_upload: keyword_upload
+        )
+
+      result = SearchResults.group_by_adword_types(search_results)
+
+      assert Map.has_key?(result, :adwords) == false
+      assert Map.has_key?(result, :top_adwords) == false
+      assert Enum.count(result.non_adwords) == 2
+    end
+
+    test "given SearchResultUrls with one URL that is an adword, returns a Map with one adword with the key adwords" do
+      keyword_upload = insert(:keyword_upload)
+
+      search_results =
+        insert_list(1, :search_result_url,
```

Comment:
when we insert `1` item, should we just use `insert` instead of `insert_list(1,...`

Reply:
I used `insert_list` because it returned it inside a List. I did this because the function `group_by_adword_types` requires a List.

I could insert only one but then I would have to wrap it inside a List anyway. 

What do you think?

Reply:
ah i see, either way work for me in this case.